### DISCUSSION TITLE
Temperature Propagation Along The Geometries

### DISCUSSION THREAD

**Discussion by SXFire12** [firemodels/cfast#2064] [Q&A]
Hello everyone, 

I'm trying to simulate complex geometries. The fire is assigned to room 13, which the Green Line indicates. An opening connects room 13 to room 1 (Red Line). 

Some notes: - I can't assign an opening between room 13 and room 1 directly. Instead, I connected room 13 to the adjacent Room. 
- The visualization before I run it; the opening is okay.
- Once I run the simulation, the temperature doesn't propagate to the room 1. Instead, it propagates to the adjacent rooms. 
- To make my question understandable, I also attached the file. 

Do you have any recommendations to overcome this issue? Shall I define room 1 into multiple rooms?

![CFAST1](https://github.com/firemodels/cfast/assets/100284932/6c626832-99fc-421d-bd50-e42f1c4a7e1e)
![CFAST2](https://github.com/firemodels/cfast/assets/100284932/f544b54a-cc00-4999-a509-e08e223a0766)
[Rooms.zip](https://github.com/firemodels/cfast/files/13318962/Rooms.zip)

**Comment by drjfloyd** [comment]
In your input file the fire is in room 13 and the only vent for room 13 is to room 14.  Room 14 has no vents to anything else. If I delete all the compartments except for 13 and 14 and run the case I see the image below.  Room 14 is the small room in the corner of room 13. The OFFSET you have defined for the vent makes it look like it connects to the hallway but it doesn't. It is connected to room 13. OFFSET is just used by Smokeview so that doors can be visualized in the correct location. It doesn't change the compartments connected.


![Rooms2_0012](https://github.com/firemodels/cfast/assets/12799217/8584c285-da88-4f81-a03d-f3e8945b491f)

**Reply by SXFire12** [reply]
Thanks for the quick feedback. I intend to make a vent from room 13 to the hallway actually. I tried to assign a vent from Room 13 to Room 1, but Cfast says that it should be from the lower to the higher order. I tried to make a vent from Room 13 (left side) to the outside, but the temperature doesn't propagate to the hallway. I tried to assign the geometry of room 1 after the room 13, it seems work well

**Comment by drjfloyd** [comment]
I think that error is just stating that when you define a VENT with the editor that the compartment in the first compartment column must have a lower numer than the compartment in the second, i.e. the compartment list needs to be 1,13 and not 13,1.  The order you list doesn't define the direction of the flow. The flow direction will be based on the pressures and layer temperatures in the two compartments.
